### PR TITLE
test_parallel_for_reduction_task_device.c: Fix directive

### PR DESCRIPTION
**Pull request by tob2**
Change 'target map' to 'omp target map'.

Rather obvious fix, once found.

@tmh97 @spophale @seyonglee @jrreap @AnonNick @krishols @mjcarr458 @nolanbaker31 – please review

_I had combined this PR with PR 612 if I had seen it earlier that both had the same issue; to keep a sensible subject line, I did not add the commit to the other PR._

### CODE REVIEW

**Review by nolanbaker31 — APPROVED** [review]

**Comment by tob2** [comment]
Patch ping – @spophale @seyonglee @jrreap @AnonNick @krishols @mjcarr458 – please review

_This is trivial and just adds a missing 'omp' to the pragma – it also only needs a second review. Thanks!_

**Comment by tob2** [comment]
Patch ping² – @spophale @seyonglee @jrreap @AnonNick @krishols @mjcarr458 – please review

This is trivial and just adds a missing 'omp' to the pragma – it also only needs a second review. Thanks!

**Review by tmh97 — APPROVED** [review]
good-to-go

**Pull request merged**
